src/components/Paragraph/index.svelte.d.ts

```diff
@@ -0,0 +1,18 @@
+import { SvelteComponentTyped } from 'svelte';
+export default class Index extends SvelteComponentTyped<IndexProps, IndexEvents, IndexSlots> {
+}
```

Comment:
do you extend with nothing? `{ .. }` is empty

Reply:
@zolotokrylin The same here